packages/p2p/src/components/modal-manager/modals/ad-error-tooltip-modal/ad-error-tooltip-modal.jsx

```diff
@@ -4,26 +4,131 @@ import { observer } from 'mobx-react-lite';
+                        }}
+                    />
+                );
+            case 'advertiser_ads_paused':
+                return (
+                    <Localize i18n_default_text='This ad is not listed on Buy/Sell because you have paused all your ads.' />
+                );
+            case 'advertiser_balance':
+                return (
+                    <Localize
+                        i18n_default_text='This ad is not listed on Buy/Sell because its minimum order is higher than your Deriv P2P available balance ({{balance}} {{currency}}).'
+                        values={{
+                            balance: general_store.advertiser_info.balance_available,
+                            currency: account_currency,
+                        }}
+                    />
+                );
+            case 'advertiser_daily_limit':
+                return (
+                    <Localize
+                        i18n_default_text='This ad is not listed on Buy/Sell because its minimum order is higher than your remaining daily limit ({{remaining_limit}} {{currency}}).'
+                        values={{
+                            remaining_limit: my_ads_store.advert_details?.max_order_amount_limit_display,
+                            currency: account_currency,
+                        }}
+                    />
+                );
+            case 'advertiser_temp_ban':
+                return (
+                    <Localize i18n_default_text='You’re not allowed to use Deriv P2P to advertise. Please contact us via live chat for more information.' />
+                );
+            default:
+                return <Localize i18n_default_text='Your ad is not listed' />;
+        }
+    };
+
+    const getMultipleErrorMessages = error_statuses =>
+        error_statuses.map((status, index) => (
+            <div key={index}>
+                {index + 1}. {getAdErrorMessage(status)}
+            </div>
+        ));
+
+    return (
+        <Modal className='p2p-my-ads__modal-error' is_open={is_modal_open} small has_close_icon={false}>
+            <Modal.Body>
+                <Text as='div' color='prominent' size={isMobile() ? 'xxs' : 'xs'} line_height={isMobile() ? 'l' : 'xl'}>
+                    {visibility_status.length === 1 ? (
+                        getAdErrorMessage(visibility_status[0])
+                    ) : (
+                        <div>
+                            <Localize i18n_default_text='Your ad isn’t listed on Buy/Sell due to the following reason(s):' />
+                            {getMultipleErrorMessages(visibility_status)}
+                        </div>
+                    )}
                 </Text>
             </Modal.Body>
             <Modal.Footer>
-                <Button has_effect text={localize('Ok')} onClick={() => hideModal()} primary large />
+                <Button has_effect text={localize('OK')} onClick={() => hideModal()} primary large />
```

Comment:
```suggestion
                <Button has_effect text={localize('OK')} onClick={hideModal} primary large />
```

Can it be like this?